harmoni_detectors/harmoni_stt/src/w2l_service.py

```diff
@@ -33,12 +34,14 @@ def __init__(self, name, param):
             )
         self.w2l_bin = param["w2l_bin"]
         self.service_id = HelperFunctions.get_child_id(self.name)
+        self.w2l_process = None
         """Setup publishers and subscribers"""
         rospy.Subscriber(
-            RouterSensor.microphone.value + self.subscriber_id + "/talking",
+            RouterSensor.microphone.value + self.subscriber_id,
             AudioData,
             self.callback,
         )
+        rospy.Subscriber("/audio/audio", AudioData, self.pause_back)
```

Comment:
"/audio/audio" seems like a poor topic name. Could remap it in the launch files.

Reply:
This is the ROS default from the audio_common library.